From cf5f65e4441b3acb94f895cb65bbc948c9101384 Mon Sep 17 00:00:00 2001
From: Konstantin Khlebnikov <khlebnikov@openvz.org>
Date: Mon, 8 Sep 2008 14:15:06 +0400
Subject: [PATCH] MISC autofs fix default pgrp vnr

Default pgrp should be virtual-nr,
because autofs lookup pid struct via find_get_pid.
---
 fs/autofs/inode.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/fs/autofs/inode.c b/fs/autofs/inode.c
index 45f5992..af82143 100644
--- a/fs/autofs/inode.c
+++ b/fs/autofs/inode.c
@@ -80,7 +80,7 @@ static int parse_options(char *options, int *pipefd, uid_t *uid, gid_t *gid,
 
 	*uid = current->uid;
 	*gid = current->gid;
-	*pgrp = task_pgrp_nr(current);
+	*pgrp = task_pgrp_vnr(current);
 
 	*minproto = *maxproto = AUTOFS_PROTO_VERSION;
 
-- 
1.5.4.3

